examples/first-component/.gitignore

```diff
@@ -22,3 +22,7 @@ dist-ssr
 *.njsproj
 *.sln
 *.sw?
+
+*storybook.log
+storybook-static
+chromatic.config.json
```

Comment:
Why we list `chromatic.config.json` into `.gitignore`

Reply:
Fixed